lib/autoload/kuroko2/command/executor.rb

```diff
@@ -19,7 +19,8 @@ def initialize
                    elsif worker_id == (Command::Executor.num_workers - 1)
                      Command::Monitor.new(hostname: @hostname, worker_id: worker_id)
                    else
-                     @worker = Worker.where(hostname: @hostname, worker_id: worker_id, queue: @queue).first_or_create!
+                     @worker = Worker.where(hostname: @hostname, worker_id: worker_id, queue: @queue).first_or_initialize!
+                     @worker.update_column(:suspendable, true)
```

Comment:
`first_or_initialize!` method doesn't exist.

Reply:
Thanks, I fixed in https://github.com/cookpad/kuroko2/pull/94/commits/34839bc21f4654f15945c058afb61df837413073.